File: loianeTraining/javaBasico/exercicios-aulas-14-15/Ex12.java
import java.util.Scanner;

class Ex12 {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite o valor do Salario Bruto");
		double grossSalary = scan.nextDouble();

		int porcentIR = 0;
		double irValue = 0;

		double syndicate = grossSalary * 0.03;
		double inss = grossSalary * 0.1;
		double fgtsValue = grossSalary * 0.11;

		if (grossSalary > 900 && grossSalary <= 1500) {
			porcentIR = 5;
			irValue = grossSalary * porcentIR / 100;
		} else if (grossSalary > 1500 && grossSalary <= 2500) {
			porcentIR = 10;
			irValue = grossSalary * (porcentIR / 100);
		} else {
			porcentIR = 20;
			irValue = grossSalary * (porcentIR / 100);
		}

		double totalDiscount = irValue + inss + syndicate;
		double netSalary = grossSalary - totalDiscount;

		System.out.println("Salario Bruto:\t: R$ " + grossSalary);
		System.out.println("(-)IR (" + porcentIR + "%)\t: R$ " + irValue);
		System.out.println("(-) INSS (10%)\t: R$ " + inss);
		System.out.println("FGTS (11%)\t: R$ " + fgtsValue);
		System.out.println("Total de desc\t: R$ " + totalDiscount);
		System.out.println("Salario Liquido\t: R$ " + netSalary);

		scan.close();
	}
}

File: loiane-training-csharp/procedural-poo/c/Ex12.cs
using System;

public class Ex12
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite o valor de sua hora:\t");
        double SalaryPerHour = double.Parse(Console.ReadLine());

        Console.WriteLine("Digite a quantidade de horas trabalhadas:\t");
        double HourWorkedPerMonth = double.Parse(Console.ReadLine());

        double GrossSalary = SalaryPerHour * HourWorkedPerMonth;
        double IRPercent = 0;
        double INSS = 10;
        double Syndicate = 3;
        double FGTS = 0.11;
        double DiscountIR = 0;

        if (GrossSalary <= 900)
        {
            DiscountIR = 0;
        }
        else if (GrossSalary > 900 && GrossSalary <= 1500)
        {
            IRPercent = 5;
            DiscountIR = 0.05 * GrossSalary;
        }
        else if (GrossSalary > 1500 && GrossSalary <= 2500)
        {
            IRPercent = 10;
            DiscountIR = 0.10 * GrossSalary;
        }
        else
        {
            IRPercent = 20;
            DiscountIR = 0.20 * GrossSalary;
        }

        double INSSDiscount = GrossSalary * 0.10;
        double TotalDiscount = DiscountIR + INSSDiscount + Syndicate;

        Console.WriteLine("Salario Bruto (" + SalaryPerHour + " * " + HourWorkedPerMonth + " ):\tR$" + GrossSalary);
        Console.WriteLine("( - )IR(" + IRPercent + "%):\t\t\tR$" + DiscountIR);
        Console.WriteLine("( - )INSS:\t\t\tR$" + INSSDiscount);
        Console.WriteLine("FGTS (" + FGTS + "%):\t\t\tR$" + (GrossSalary * 0.11));
        Console.WriteLine("Total de descontos:\t\tR$" + TotalDiscount);
        Console.Write("Salario Liquido: \t\tR$" + (GrossSalary - TotalDiscount));
    }
}
